### ISSUE TITLE
richtige Belegung für Shifter G27

### ISSUE THREAD

**Issue by skull8** [modelleicher/FS19_realManualTransmission#4]
Moin,

erstmal danke für deine Arbeit.
Ich stehe momentan nur vor einem Problem: die Gänge kann ich einzeln anwählen, jedoch wie hast du dir das mit Rückwärtsgang gedacht? Da ich über meinen Shifter jeden Gang einzeln anwählen kann, würde ich das gerne auch mit dem Rückwärtsgang machen, jedoch funktioniert das in keiner Belegung.
Entweder fehlt eine direkte Anwahl, oder ich bin zu doof dein System zu belegen. :D

**Comment by modelleicher** [comment]
Moin,

Es gibt keine direkte Anwahl für den Rückwärtsgang, sofern dieser nicht zu den Gängen gehört.

Das System ist so aufgebaut wie in echt, mit der Gangwahl kannst du bis zu 8 Gänge anwählen. Die Gruppen allerdings nicht, oder die Reverser.
Somit wenn der Traktor in echt einen Rückwärtsgang hat kann man diesen auch auf der entsprechenden Belegung anwählen (entweder auf 1 oder eben auf max. Vorwärts +1, je nach dem wo der R-Gang in echt eben ist) 

Hat der Traktor den Rückwärtsgang jetzt in einer Gruppe dann kommst du da nicht über die Gangwahl dran, das selbe gilt natürlich für den Reverser der ist ja in echt i.d.R. auch ein kleiner Hebel oder Knopf neben dem Lenkrad ;) 

LG

**Comment by skull8** [comment]
Wie es in echt ist, weiss ich. Man ist halt etwas eingeschränkt am PC. Da muss ich mir was mit dem Reverser überlegen. Danke erstmal.

**Comment by modelleicher** [comment]
Du kannst ja die Reverser-Umschalten Taste mit dem R-Gang des G27 belegen zum Beispiel.

LG

**Comment by skull8** [comment]
Eben das hatte nicht funktioniert, aber ich habe da eine Vermutung, dass es sich mit einem anderen Mod beißt. (Vehicle Enhanced). Das will ich morgen nochmal überprüfen.

**Comment by modelleicher** [comment]
Ja, Doppelbelegungen aufm gleichen Fahrzeug sind ein Problem.. Hoffe das wird mit dem nächsten Patch gefixt.
Das scheint aber als Workaround zu funktionieren:
https://github.com/modelleicher/FS19_realManualTransmission/issues/3#issuecomment-460706407


LG
